.gitignore

```diff
@@ -12,3 +12,4 @@ examples/**/.draftignore
 pkg/linguist/data/classifier
 pkg/linguist/data/linguist
 pkg/linguist/data/*.yml
+.idea/
```

Comment:
Hey! Generally speaking we tend to keep IDE-specific stuff outside of .gitignore and in a (personal) global gitignore. e.g. on my system, all I have is:

```
><> git config --get core.excludesfile
~/.gitignore_global
><> cat ~/.gitignore_global 
.DS_Store
.vscode/
```

This allows us to keep the .gitignore specific to anything the project uses, and files that the OS or IDE generates up to the user.

With that in mind, would you mind removing this? :)

Reply:
sure thing, done.